xunit.runner.visualstudio.nuspec

```diff
@@ -41,5 +41,8 @@ Supported platforms:
     <file src="build\xunit.runner.visualstudio.wpa81.targets"                                   target="build\wpa81\xunit.runner.visualstudio.targets" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.dll" target="build\wpa81\" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.pri" target="build\wpa81\" />
+
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.deps.json" target="build\netstandard1.5\" />
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.dll" target="build\netstandard1.5\" />
```

Comment:
I'm not sure the logic about PCL's holds true for .NET Standard.... but as you say, it's philosophical. If I'm writing a .NET Standard 1.3 class library, I would expect to write a .NET Standard 1.3 test class library. Under the covers, I shouldn't care whether it runs on .NET Core or desktop, as those differences are supposed to be hidden from me. Of course, I realize that in reality there should not be differences in the contract surface area.

I suspect most libraries should be fine testing their logic using a .net standard test library though. Anything else is really testing the platform.